src/kOS.Safe/Compilation/KS/Compiler.cs

```diff
@@ -606,6 +606,9 @@ private void VisitNode(ParseNode node)
                 case TokenType.arglist:
                     VisitArgList(node);
                     break;
+				case TokenType.hudtxt_stmt:
+					VisitHudtxtStatement(node);
+					break;
```

Comment:
There's a common trend where programmers tend to think of the "tab" ascii character as a software re-assignable width that can be whatever you like, and that tends to mess up when different people have it set to different values in their viewers.  And in all cases where it's hardcoded and can't be changed, it's 8 columns, and thus picking a value of 4 makes it come out wrong in those places.

To get around this problem we've made the standard that the source files just shouldn't have any tabs at all.  It makes it simpler to deal with.  Most editors can be made to export the file with tabs turned into runs of spaces.

It's easy enough to fix for this pull request.  Just something to remember if you want to make more later.